requirements-cpu.conda

```diff
@@ -2,7 +2,7 @@ python>=3.7,<3.8a0
 colorama
 numpy
 opt_einsum
-abergeron::tvm==0.7dev1+1.*
+notoraptor::tvm==0.7dev1+1.*
```

Comment:
You need to change the version if you change the code (the +1 should be +2).  Your +1 should be the same as my +1.

Reply:
By the way, this is changes applied to tvm: https://github.com/apache/incubator-tvm/compare/master...notoraptor:fix-excepthook

Should I still recompile and rename my package to ` notoraptor::tvm==0.7dev1+2.*` ?

Reply:
Yes, otherwise it can easily create problems where people don't know which version of the code they have exactly and it makes it harder to debug problems.

This is why it's not a good practice to release two packages with the same version and different code.

Reply:
Ok, so I will recompile and re-upload a tvm package with version `+2`!